src/widgets/input/MultiInputWidget.js

```diff
@@ -0,0 +1,122 @@
+// @flow
+import React, { useState } from 'react';
+
+import styled from 'styled-components';
+import { Input } from 'lattice-ui-kit';
+
+import type { WidgetProps } from '../../../types';
+
+const InputWrapper = styled.div`
+  display: grid;
+  grid-template-columns:${(props) => `${props.flexOptions[0]}fr ${props.flexOptions[1]}fr`};
+  grid-gap: 20px;
+  margin-bottom: 15px;
+  align-items: center;
+
+  >span {
+    font-size: 14px;
+  }
+`;
+
+const MultiInputWidget = (props :WidgetProps) => {
+
+  const {
+    autofocus,
+    disabled,
+    formContext,
+    id,
+    onBlur,
+    onFocus,
+    options,
+    rawErrors,
+    readonly,
+    registry,
+    schema,
+    value,
+    onChange,
+    ...inputProps
+  } = props;
+
+  inputProps.type = 'text';
```

Comment:
inputProps.type can sometimes be a number, as your example schema suggests. you should use this to help enforce html attributes on the input. Inputs that are numbers or integers should adhere to min/max/step attributes. Should let the browser do its default number behavior as shown below (increment arrows with keyboard support)

![image](https://user-images.githubusercontent.com/27182199/93384706-0da55a80-f81a-11ea-8449-cf9a4789a341.png)
https://openlattice.github.io/lattice-fabricate/?path=/story/widgets--number

Reply:
Good point. Thanks!